Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using PatchKit.Unity.Patcher;
+using PatchKit.Unity.Patcher.Debug;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit.Unity.Patcher.Debug
+{
+    public class DebugMenu : MonoBehaviour
+    {
+        private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(DebugMenu));
+
+        private Rect _popupRect, _rect;
+        private bool _showDebugMenu;
```

Comment:
Please rename to `_show` (my last comment might have been misleading as it was already named `_show`).

Reply:
done